assignments/lambda-classes.js

```diff
@@ -1 +1,95 @@
 // CODE here for your Lambda Classes
+class Person {
+    constructor(props){
+        this.name = props.name;
+        this.age = props.age;
+        this.location = props.location;
+    }
+    speak() {
+        return(`Hello, my name is ${this.name} and I'm from ${this.location}`);
+    }
+}
+
+class Instructor extends Person {
+    constructor(instructorArrts){
+        super(instructorArrts);
+            this.specialty = instructorArrts.specialty;
+            this.FavLanguage = instructorArrts.favLanguage;
+            this.catchPhrase = instructorArrts.catchPhrase;
+            this.channel = instructorArrts.channel;
+    }
+    demo(subject){
+        return(`Today we are learning about ${subject}`);
+    }
+    grade(student, subject){
+        return(`${student.name} receives a perfect score on ${subject}`);
+    }
+}
+class Student extends Person {
+    constructor(studentArrts){
+        super(studentArrts);
+        this.previousBackground = studentArrts.previousBackground;
+        this.className = studentArrts.className;
+        this.favSubjects = studentArrts.favLanguage;
```

Comment:
here you should have put: this.favSubjects = studentsArrts.favSubjects;  

notice the pattern? they are always named the same after the dot!